Sources/Kitura/RouterRequest.swift

```diff
@@ -42,12 +42,12 @@ public class RouterRequest {
 
     /// The server request.
     let serverRequest: ServerRequest
-    
-    /// The Data decoder generator for the request content-type
-    let decoder: BodyDecoder?
 
     // MARK: Properties
     
+    /// The BodyDecoder that will be used to decode Data into a Codable type.
+    public var decoder: BodyDecoder?
```

Comment:
These properties only really need to be publicly settable - would it be more appropriate to keep them internal, but provide a public setter?